examples/function_node/main.tf

```diff
@@ -13,15 +14,226 @@ provider "aci" {
   insecure = true
 }
 
-resource "aci_function_node" "foofunction_node" {
+data "aci_tenant" "tf_tenant" {
+  name = "tf_ansible_test"
+}
+data "aci_vrf" "tf_vrf" {
```

Comment:
make the spacing consistent (have a blank line between each data source or resource)

Reply:
changed